acestep/models/xl_turbo/modeling_acestep_v15_xl_turbo.py

```
@@ -2214,6 +2215,36 @@ def generate_audio(
+                        step_index=step_idx,
+                        attention_mask=attention_mask,
+                        repaint_mask=repaint_mask,
+                    )
+                    if corrected.shape == xt.shape and torch.isfinite(corrected).all():
+                        xt = corrected.detach()
+                    else:
+                        logger.warning(
+                            "Topology corrector returned an invalid tensor at step %d; using no-op.",
+                            step_idx + 1,
+                        )
+                except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)